Scripts/Python/QueryGroupManagementOmeREDFISH.py

```diff
@@ -0,0 +1,706 @@
+#
+# QueryGroupManagementOmeREDFISH. Python script using Redfish API with OEM extension to manage Open Manage Enterprise Query Groups
+#
+# _author_ = Texas Roemer <Texas_RoemerDell.com>
+# _version_ = 1.0
+#
+# Copyright (c) 2020, Dell, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+#
+
+
+
+
+import requests, json, sys, re, time, warnings, argparse, os
+
+from datetime import datetime
+
+warnings.filterwarnings("ignore")
+
+parser = argparse.ArgumentParser(description='Python script using Redfish API with OEM extension to create/update/delete Open Manage Enterprise Query Groups')
+parser.add_argument('-ip', help='OME IP Address', required=True)
+parser.add_argument('-u', help='OME username', required=True)
```

Comment:
Could you please rework this like the other scripts - there is a standard template to add in the help text / examples using the RawTextHelpFormatter - this would be consistent with other scripts in the site